bundles/framework/coordinatetool/plugin/CoordinatePluginHandler.js

```diff
@@ -83,28 +84,29 @@ class UIHandler extends StateHandler {
 
     setLon (value) {
         this.updateState({
-            displayXy: {
-                lonlat: {
-                    lat: this.state.displayXy?.lonlat?.lat,
-                    lon: value
-                }
-            }
+            lonField: value
         });
         this.updatePopup();
     }
 
     setLat (value) {
         this.updateState({
-            displayXy: {
-                lonlat: {
-                    lat: value,
-                    lon: this.state.displayXy?.lonlat?.lon
-                }
-            }
+            latField: value
         });
         this.updatePopup();
     }
 
+    async blur () {
```

Comment:
We should rename this from "when this is called" to something more descriptive like "what happens when this is called". Maybe `useUserDefinedCoordinates()`?